tests/Microsoft.Identity.Test.Unit/BrokerTests/BrokerRequestTests.cs

```diff
@@ -280,6 +280,7 @@ public void GetAccountWithDuplicateBrokerAccountsTest()
             broker.IsBrokerInstalledAndInvokable().Returns(true);
 
```

Comment:
`IosBrokerMock` needs to return `false` for `CanBrokerSupportSilentAuth`.
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/5da17a099ef18727f821aa13e3f8a426f0578a49/tests/Microsoft.Identity.Test.Unit/BrokerTests/BrokerRequestTests.cs#L295

Reply:
Sorry, I actually misspoke there. 
`IosBrockerMock.IsBrokerInstalledAndInvokable` should return `true`. 

And in this test, method, we should add `platformProxy.CanBrokerSupportSilentAuth().Returns(false);`to represent the iOS scenario correctly.
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/5da17a099ef18727f821aa13e3f8a426f0578a49/tests/Microsoft.Identity.Test.Unit/BrokerTests/BrokerRequestTests.cs#L415

Reply:
>> IosBrockerMock.IsBrokerInstalledAndInvokable should return true.
Done
>> And in this test, method, we should add platformProxy.CanBrokerSupportSilentAuth().Returns(false);
Done
>> Additionally we need a test for a scenario when iOSBroker.IsBrokerInstalledAndInvokable actually throws that exception
iOSBroker inherits from NSObject hence has a dependency on iOS. So, it cannot be tested here.